lib/posix/key.c

```diff
@@ -120,6 +121,8 @@ int pthread_key_create(pthread_key_t *key,
  */
 int pthread_key_delete(pthread_key_t key)
 {
+	size_t bit;
+	__unused int ret;
```

Comment:
moving the `__unused` to the right can workaround the checkpatch, or maybe add `__unused` to the attribute list?

https://github.com/zephyrproject-rtos/zephyr/blob/69d62add980bcafa6d90b73920372390cc10ea7c/scripts/checkpatch.pl#L379-L405

Reply:
merged, good stuff